lntest/itest/lnd_test.go

```diff
@@ -1344,14 +1344,14 @@ func basicChannelFundingTest(t *harnessTest, net *lntest.NetworkHarness,
 	}
```

Comment:
Could you also add tests that check the new `ChannelBalance` fields too? (Including the ones that became deprecated, to ensure we still are backwards compatible.)

Reply:
sure, test added.